digdag-core/src/main/java/io/digdag/core/agent/ConfigEvalEngine.java

```diff
@@ -63,6 +63,7 @@ private static String readResource(String resourceName)
 
     private final ObjectMapper jsonMapper;
     private final NashornScriptEngineFactory jsEngineFactory;
+    private DigdagMetrics metrics;
```

Comment:
this variable is not used. please delete it as well as `import` lines

Reply:
Thanks. Removed it.